### ISSUE TITLE
This Issue is about the  rectangle form debug in p2 physic.

### ISSUE THREAD

**Issue by spiritabsolute** [photonstorm/phaser-ce#258]
This Issue is about the  rectangle form debug in p2 physic.

If I rotate the rectangle form, then in debug mode the boundary of the collider does not rotate, it is displayed incorrectly.

* A bug in the API:
  * Phaser version(s): 2.8.0
  * Live example: http://phaser.io/sandbox/MKAwWVbo/play
  * What should happen: The rectangle should be displayed at an angle.
  * What happens instead: The rectangle is displayed without inclination angle.

```
game.physics.startSystem(Phaser.Physics.P2JS);
    var sprite = game.add.sprite(100, 100);
    game.physics.p2.enable(sprite, true);
    sprite.body.setRectangle(20, 90, -30, -30, `game.math.degToRad(80));
```

**Comment by samme** [comment]
Can you try copying <https://samme.github.io/phaser-examples-mirror/p2%20physics/body%20debug.html> and rotate the block?

**Comment by spiritabsolute** [comment]
sprite.body.rotateLeft(2); work fine, but i need .setRectangle rotate. I have static body. I want to turn the box  through .setRectangle and see the result of the turn.

**Comment by samme** [comment]
Assume fixed by b1db282.
